packages/registry/docs/howto/how-to-initialize-registry.md

```diff
@@ -0,0 +1,6 @@
+### Initialization
+
+% QUESTION: Should this go to "HowTo"s?
```

Comment:
It does not yet qualify as a how-to. In its current state, it is an explanation or conceptual guide.